opentrons-ai-client/src/atoms/AttachedFileItem/AttachedFileItem.module.css

```
@@ -0,0 +1,99 @@
+@import '@opentrons/components/styles';
+
+.container {
+  position: relative;
+  display: flex;
+  align-items: flex-start;
+  gap: var(--spacing-8);
+  padding: var(--spacing-8);
+  background-color: rgba(22, 33, 45, 0.2);
+  border-radius: var(--border-radius-8);
+  border: none;
+  width: fit-content;
+  min-width: 10rem;
+  max-width: 100%;
+}
+
+.container_with_remove {
+  padding-right: var(--spacing-40);
+}
+
+.container_without_remove {
+  padding-right: var(--spacing-8);
+}
+
+.remove_button {
+  position: absolute;
+  top: var(--spacing-2);
+  right: var(--spacing-2);
+  background: none;
+  border: none;
+  cursor: pointer;
+  padding: var(--spacing-2);
+  border-radius: var(--border-radius-4);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: var(--spacing-32);
+  height: var(--spacing-32);
```

--spacing-number is for padding, margin, gap etc.
so let's use rem for width and height.


```suggestion
  width: 2rem
  height: 2rem;
```

---

got it!